src/ray/rpc/worker/core_worker_server.h

```diff
@@ -16,12 +16,12 @@ namespace rpc {
 /// NOTE: See src/ray/core_worker/core_worker.h on how to add a new grpc handler.
 #define RAY_CORE_WORKER_RPC_HANDLERS                                          \
   RPC_SERVICE_HANDLER(CoreWorkerService, AssignTask, 5)                       \
-  RPC_SERVICE_HANDLER(CoreWorkerService, PushTask, 9999)                      \
+  RPC_SERVICE_HANDLER(CoreWorkerService, PushTask, 100)                       \
   RPC_SERVICE_HANDLER(CoreWorkerService, DirectActorCallArgWaitComplete, 100) \
-  RPC_SERVICE_HANDLER(CoreWorkerService, GetObjectStatus, 9999)               \
-  RPC_SERVICE_HANDLER(CoreWorkerService, WaitForObjectEviction, 9999)         \
-  RPC_SERVICE_HANDLER(CoreWorkerService, WaitForRefRemoved, 9999)             \
-  RPC_SERVICE_HANDLER(CoreWorkerService, KillActor, 9999)                     \
+  RPC_SERVICE_HANDLER(CoreWorkerService, GetObjectStatus, 100)                \
+  RPC_SERVICE_HANDLER(CoreWorkerService, WaitForObjectEviction, 100)          \
```

Comment:
I think this is actually dangerous to have so low - if a worker owns more than 100 objects things can go wrong because the 101th+ request won't be processed at all